docs/dev-manuals/architecture/services.md

```diff
@@ -0,0 +1,76 @@
+# Services
+
+1. Product
+   - stores concrete product information - name, description, image locations, and product variants, i.e. XXL T-Shirt in red - for each product
+   - does not store individual instances but the overall descriptions of an item. For that see `inventory`
+   - basepath `/products`
+   - CRUD on `/products`, CRUD on `/products/<product>/variants`
+1. Account
+   - Handles session information for a user, including login/registration
+   - In our case, most likely a Keycloak
+   - basepath `/account`
+   - routes `/account/register`, `/account/login`, \<TODO: Path to query JWT token of a user\>
+1. User
+   - user settings (whenJoined, birthday, gender)
+   - CRUD on `/user/<username>/settings`
+1. Notification
+   - Provides notfications to show a user
+   - mail replacement
+   - supposed to cover everything found under a typical bell icon
+   - basepath `/notifications`
+   - routes `/user/<username>/notifications?state=<state>`, `/user/<username>/notifications/<notification>/change-read`
+1. Address
+   - Stores possible addresses, and to which user they belong
+   - routes `/user/<username>/addresses`
```

Comment:
Addresses should be queried without a reference to the user, as they are used for orders/returns, etc. I would suggest an ID:

```suggestion
   - routes `/address/<id>`, `/addresses`
```